pages/common/miniserve.md

```diff
@@ -0,0 +1,16 @@
+# miniserve
+
+> Simple HTTP file server CLI.
+> More information: <https://github.com/svenstaro/miniserve>.
+
+- Serve a directory:
```

Comment:
Is `serve` a common word? I assume it means something like `make accessible`.
What do you think about
```suggestion
- Make a directory accessible over HTTP:
```

Reply:
Yeah, `Serve` is a commonly used word for this. In fact, there's a Node.js package called `serve` that does something very similar to this command.